storage/operations.go

```diff
@@ -68,6 +68,15 @@ func DefaultIteratorOptions() IteratorOption {
 	}
 }
 
+// Seeker is an interface for seeking keys in a given range.
+type Seeker interface {
+	// SeekLE (seek less than or equal) returns given key if present.  Otherwise,
```

Comment:
Can we also add some comments for the `bool` return value.

Reply:
> Can we also add some comments for the bool return value.

I removed the bool return value while simplifying `SeekLE` API.  Also updated docs.